webapp/src/js/controllers/reports.js

```diff
@@ -255,8 +255,10 @@ angular
     };
 
     $scope.search = function() {
-      $state.go('reports.detail', { id: null }, { notify: false });
-      clearSelection();
+      if($scope.filters.search) {
```

Comment:
`search` is for freetext search - does this also clear the selection when using the filters?

Reply:
Have updated to include the rest of the filters.